docs/automated-testing/e2e-testing/README.md

```diff
@@ -137,7 +137,6 @@ Run tests in parallel or on a Selenium Grid with built-in Selenium WebDriver. Ra
 
 **Ranorex Studio** tests also integrate with Azure DevOps (AzDO), which can be run as part of a build pipeline in AzDO.
 
-[Ranorex Studio Website](https://www.ranorex.com/free-trial/?utm_source=Google&utm_medium=cpc&utm_id=7015a000001ohs3AAA&utm_campaign=14484569941&utm_source=g_c&utm_medium=cpc&utm_content=588982669352&utm_term=p_ranorex%20studio&adgroupid=132316228251&gclid=EAIaIQobChMIsoqzgdzD-AIVygMGAB1pHQrpEAAYASAAEgKed_D_BwE)
```

Comment:
GENERAL RECOMMENDATION: remove files having changes due exclusively to auto-formatting/whitespace edits from this PR and submit as part of a separate PR (focused on misc. whitespace/formatting updates) in order to reduce the 'noise' in this PR.

Reply:
This is fixed by latest merge from main branch.